cpp/data/system_updates/12.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -o errexit
+
+
+if [[ $TUEFIND_FLAVOUR == "ixtheo" ]]; then
+    if [ -e /var/spool/cron/root ]; then # CentOS
+        root_crontab=/var/spool/cron/root
+    elif [ -e /var/spool/cron/crontabs/root ]; then # Ubuntu
+        root_crontab=/var/spool/cron/crontabs/root
+    else
+        exit 0 # There is no crontab for root.
+    fi
+
+    sed --in-place '/^# END VUFIND AUTOGENERATED/i 30 0 * * * "$BIN/generate_new_journal_alert_stats ixtheo days_in_last_month"\n30 0 * * * "$BIN/generate_new_journal_alert_stats relbib days_in_last_month"' $root_crontab
```

Comment:
This does not insert the new cronjob at the same position as in ixtheo.cronjobs, but it should do so.

Reply:
Done!